samples/contrib/e2e-mnist/mnist-pipeline.ipynb

```diff
@@ -42,14 +66,18 @@
     "\n",
     "    return \" \".join(args)\n",
     "\n",
+    "def add_istio_annotation(op):\n",
+    "    op.add_pod_annotation(name='sidecar.istio.io/inject', value='false')\n",
+    "    return op\n",
+    "\n",
     "@dsl.pipeline(\n",
     "    name=\"End to end pipeline\",\n",
     "    description=\"An end to end example including hyperparameter tuning, train and inference.\"\n",
     ")\n",
     "def mnist_pipeline(\n",
-    "        name=\"mnist-{{workflow.uid}}\",\n",
-    "        namespace=\"kubeflow\",\n",
-    "        step=2000):\n",
+    "        name=model_name,\n",
+    "        namespace=user_namespace,\n",
+    "        step=4000):\n",
```

Comment:
Can we define what these steps signify somewhere in DSL?

Reply:
I added a markdown cell to explain each parameter right before this pipeline because comments in the DSL is a little hard to tell since the original pipeline is very complex.